cdap-ui/app/directives/program-status/program-status.js

```diff
@@ -0,0 +1,34 @@
+angular.module(PKG.name + '.commons')
+  .directive('myProgramStatus', function () {
+
+    return {
+      restrict: 'E',
+      scope: {
+        type: '@',
+        runid: '@'
+      },
+      templateUrl: 'program-status/program-status.html',
+      controller: function($scope, MyDataSource, $state) {
+        if ($scope.runid !== 'No Runs!') {
```

Comment:
Can we use $scope.runs.length property? At some point in time if we want to change the label for No Runs! to something else then this reference needs to be changed.